entity/src/product.rs

```diff
@@ -9,6 +9,7 @@ pub struct Model {
     pub id: Uuid,
     pub name: String,
     pub vendor_id: Option<Uuid>,
+    pub cpe_key: Option<String>,
```

Comment:
have you considered using **cpe** crate (we already have in Cargo.toml)  to also validate cpe with `Cpe::parse(cpe_str)?;` somewhere ?